app/javascript/controllers/work_type_controller.js

```diff
@@ -18,7 +18,13 @@ export default class extends Controller {
     // icon, target is the span and currentTarget is the anchor.
     this.formTarget.action = event.currentTarget.dataset.destination
     if (event.currentTarget.dataset.formMethod) {
-      this.formTarget.method = event.currentTarget.dataset.formMethod
+      if (this.formTarget.dataset.turbo === 'false') {
+        // If this is a non-Turbo form, we need to set the method using a hidden input
```

Comment:
I didn't realize that `_method` could be used this way on the Rails side. Thanks!

https://guides.rubyonrails.org/form_helpers.html#how-do-forms-with-patch-put-or-delete-methods-work-questionmark